samples/materials/image.mat

```diff
@@ -9,6 +9,14 @@ material {
             type : mat3,
             name : transform,
             precision : high
+        },
+        {
+            type : float3,
+            name : backgroundColor
+        },
+        {
+            type : int,
+            name : showImage
```

Comment:
We support `bool` here.

Reply:
We do but there are bugs with qcom drivers so we've been using int instead in several materials.

Reply:
Interesting -- I never knew or have completely forgotten.